.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [master, develop]
+
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Update package lists and install necessary build/test systemwide deps
+      - name: Update apt sources/Install system-wide deps
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y lua5.1 liblua5.1-0-dev luarocks git make
+
+      - name: Setup project deps
+        run: sudo make setup
+
+      - name: Download and install kong
+        run: |
+          wget -c https://bintray.com/kong/kong-community-edition-deb/download_file?file_path=dists%2Fkong-community-edition-0.13.1.xenial.all.deb -O kong.deb
```

Comment:
não é melhor chamar esse ci de ci-0.13.1 se ele é específico para essa versão do kong community edition?

Reply:
E adicionando a isso, podemos ter um CI que seja para a versão do kong que a gente precisa de fato dar suporte? 0.14.x se não me engano

Reply:
se for o caso podem ter vários arquivos desse para versões diferentes

Reply:
Feito!

Essa versão do kong é a que está setada como dependência para o template-transformer. Em alguns ambientes, temos problemas na instalação do kong via luarocks, pois ele tem uma dependência (luasec) que por sua vez depende do OpenSSL 1.0 (deprecated). A instalação do kong usando o pacote .deb contorna este problema.